### PR TITLE
rename "pml" to "PML" in Python

### PR DESCRIPTION
**Pull request by stevengj**
Following PEP 8 and discussion in #98.  cc @ChristopherHogan

### CODE REVIEW

File: python/tests/physical.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def radiating_base(self, sq_ratio, solve_cw=True):
 
         w = 0.30
 
-        s = mp.structure(self.gv, one, mp._pml(self.ymax / 3.0))
+        s = mp.structure(self.gv, one, mp.pml(self.ymax / 3.0))
```
Comment (stevengj) [inline python/tests/physical.py]: @ChristopherHogan, should this test be using `mp.PML` (i.e. the high-level object) nowadays?

File: python/tests/source.py [hunk 2]
```diff
@@ -57,7 +57,7 @@ def dummy_eps(v):
         gv = mp.voltwo(16, 16, 10)
         gv.center_origin()
         sym = mp.mirror(mp.Y, gv)
-        the_structure = mp.structure(gv, dummy_eps, mp._pml(2), sym)
+        the_structure = mp.structure(gv, dummy_eps, mp.pml(2), sym)
```
Comment (stevengj) [inline python/tests/source.py]: also `mp.PML` here?

File: python/tests/physical.py (hunk 1 above)
Comment (ChristopherHogan) [reply python/tests/physical.py]: This is a low level test, so rewriting the whole thing in the high level API is on my todo list. You can leave it as is for now.

File: python/tests/source.py (hunk 2 above)
Comment (ChristopherHogan) [reply python/tests/source.py]: This test creates low level objects to test typemaps, so I think `mp.pml` is fine.

**Pull request merged**